estate/models/estate_property.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name ="estate.property"
+    _description="test description"
+
+
+    name=fields.Char(required=True,default="Unknown")
```

Comment:
```suggestion
    name = fields.Char(required=True, default="Unknown")
```
General comment to keep in mind from next time:
Please ensure the entire module follows coding guidelines by adding spaces wherever necessary.

Reply:
ensured the coding guidelines are followed according to your suggestion.